packages/edit-site/src/store/private-actions.js

```diff
@@ -18,3 +18,10 @@ export function registerRoute( route ) {
 		route,
 	};
 }
+
+export function unregisterRoute( name ) {
```

Comment:
At the moment both of these functions are private actions. This addition is actually a bug fix because I noticed that all of our routes were duplicated in "dev" mode because React runs the effects twice.

Reply:
> we're fine with not protecting the unregistration of any core routes?

My opinion is that it's great that you can unregister and override a core route. That's how Gutenberg itself started, being able to override the core `post.php` editor page.